datacapturing/src/androidTest/java/de/cyface/datacapturing/MovebisDataCapturingService.kt

```diff
@@ -101,6 +102,7 @@ class MovebisDataCapturingService internal constructor(
     DefaultPersistenceLayer(context, CapturingPersistenceBehaviour()),
     DefaultDistanceCalculation(),
     DefaultLocationCleaning(),
+    ExampleLocationAnonymization(DefaultDistanceCalculation()) { 150.0 + Math.random() * 50.0 },
```

Comment:
The `MovebisDataCapturingService` is used to be able to test the `MovebisDataCapturingService`. The SR App implements `MovebisDataCapturingService` itself.

I've used a 150-200 m Radius as an example here.